Added prosody reservation config for docker

Mahdhir opened the pull request:
Fixes #1300

saghul commented:
Thank you! Shouldn't you also be deleting the existing reservation system config from jicofo, since it's no longer functional?

Mahdhir commented:
@saghul noted and removed the existing reservation system config from jicofo.
Also renamed the variables as they were not related Jicofo anymore.
Might need to update the documentation as well.

saghul commented:
Thank you!

Mahdhir commented:
Awesome.
@saghul  I have put a PR for the documentation as well.

shawnchin commented on prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua:
```diff
@@ -161,6 +166,10 @@ VirtualHost "{{ $XMPP_DOMAIN }}"
     {{ end }}
 
+    {{ if $PROSODY_RESERVATION_ENABLED }}
+    reservations_api_prefix = { "{{ $PROSODY_RESERVATION_REST_BASE_URL }}" }  
```
Should the curly braces around the var be there? That var is expected to be a string, not a table.

Should probably be:
```
reservations_api_prefix = "{{ $PROSODY_RESERVATION_REST_BASE_URL }}" 
```

Mahdhir replied:
Noted. Will fix it
